packages/@aws-cdk/aws-kms/test/test.key.ts

```diff
@@ -311,7 +311,7 @@ export = {
           DeletionPolicy: "Retain",
           UpdateReplacePolicy: "Retain",
         },
-        MyKeyAlias1B45D9DA: {
+        MyKeyAliasaliasxoo1B45D9DA: {
```

Comment:
Testing it locally, the logical ID suffix kept changing, e.g. `MyKeyAliasaliasxoo9464EB1C`.

Reply:
@rix0rrr Out of curiosity, what was causing my IDs to be unstable?

Reply:
Honestly, I have no idea. I just changed them to the expectations. Might it have been copy/pasting  between tests? I see some of them had similar but slightly different construct IDs.

Reply:
I ran the tests again on my original commit, and yeah they're stable. I must have misread the test output. Sorry about that!